src/UBAFeeCalculator/UBAFeeConfig.ts

```diff
@@ -102,6 +118,24 @@ class UBAConfig {
     const chainTokenCombination = `${chainId}-${tokenSymbol}`;
     return this.balanceTriggerThreshold[chainTokenCombination] ?? this.balanceTriggerThreshold;
   }
+
+  /**
+   * Get the incentive pool adjustment
+   * @param chainId The chain id
+   * @returns The incentive pool adjustment
+   */
+  public getIncentivePoolAdjustment(chainId: string): BigNumber {
+    return this.incentivePoolAdjustment[chainId];
```

Comment:
```suggestion
    return this.incentivePoolAdjustment[chainId] ?? 0;
```

Reply:
Works for me - but we may want to define a specific default constant so that we don't have multiple magic numbers